src/ui/ResultsPerPage/ResultsPerPage.ts

```diff
@@ -212,13 +213,16 @@ export class ResultsPerPage extends Component {
     this.span = $$(
       'span',
       {
+        id: `coveo-results-per-page-text-${QueryUtils.createGuid()}`,
```

Comment:
Could this instead be a simple `1`, `2`, `3` etc. for each choice ?

I'd prefer that to generating a complex guid (which is really meant for query identifier), which is a rather complex operation under the hood.

Reply:
But this `id` is not for each choice of the `ResultsPerPage`, it's for the span that holds the "Results per page" and that needs to be linked to the `ul`'s `aria-labeledby` attribute with a unique id. I think what @btaillon meant was to avoid conflicts with the ID if more than one `ResultsPerPage` is in the search interface. Maybe a GUID requires expensive computation, but otherwise having control of the number of `ResultsPerPage` instances on the interface would be required to have the sequence you mentioned. Maybe a random big int?

Reply:
@rdeandaCoveo is right, this is to avoid getting the same `id` twice in the page if there's multiple `ResultsPerPage`.

I'll generate a random `int` instead.

Reply:
UPDATE: I found out that underscore has a `uniqueId` function that will make this a lot cleaner.